fhir-helper/src/androidMain/kotlin/care/data4life/sdk/helpers/DocumentReferenceBuilder.kt

```diff
@@ -98,7 +98,7 @@ object DocumentReferenceBuilder {
         }
 
         val content = attachments.map { DocumentReference.DocumentReferenceContent(it) }
-        val document = DocumentReference(content, indexed, status, type)
+        val document = DocumentReference(status, type, indexed, content)
```

Comment:
It also changed the order into the FHIR desired one instead of alphabetical -> need to add this to the migration guide